plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeColumnHandle.java

```diff
@@ -183,4 +183,9 @@ public static DeltaLakeColumnHandle fileModifiedTimeColumnHandle()
     {
         return new DeltaLakeColumnHandle(FILE_MODIFIED_TIME_COLUMN_NAME, FILE_MODIFIED_TIME_TYPE, OptionalInt.empty(), FILE_MODIFIED_TIME_COLUMN_NAME, FILE_MODIFIED_TIME_TYPE, SYNTHESIZED);
     }
+
+    public static DeltaLakeColumnHandle getMergeRowIdColumnHandle()
```

Comment:
Rename to `mergeRowIdColumnHandle` to match the other ones of these types of methods.